components/hero.js

```diff
@@ -7,13 +7,13 @@ const Title = glamorous.h1((props, {colors, mediaQueries}) => ({
   color: colors.primaryMed,
```

Comment:
Cool, I look forward to seeing what it looks like in the deployed version (hopefully that doesn't fail again)